etc/cvmfs/config.d/atlas-nightlies.cern.ch.conf

```diff
@@ -1,6 +1,5 @@
-if [ "$CVMFS_HTTP_PROXY" = "DIRECT" ] || [ "$CVMFS_HTTP_PROXY" = "auto;DIRECT" ]; then
+if [ "$CVMFS_USE_OPENHTC" = "yes" ]; then
```

Comment:
Let's change this to CVMFS_USE_CDN as @rptaylor [suggested](https://github.com/cvmfs/cvmfs/pull/2586#issuecomment-658406333), both here, elsewhere in this PR, and [cvmfs PR 2586](https://github.com/cvmfs/cvmfs/pull/2586).